### ISSUE TITLE
Require ALT Text By Default

### ISSUE THREAD

**Issue by MichaelGHUser** [bluesky-social/social-app#1099]
**Is your feature request related to a problem? Please describe.**

ALT Text

![image](https://github.com/bluesky-social/social-app/assets/123146201/570653c2-2198-43db-bcd1-88f95bc4bbee)

**Describe the solution you'd like**

"Require alt text on images" toggle should be Enabled by default on all accounts 

**Describe alternatives you've considered**

Reminding people 

**Additional context**

Typically, I'm _not a fan_ of forcing users to opt-out of a feature, but I think this is a reasonable exception.  If inclusivity is the goal of the platform, **accessibility shouldn't be optional.**  If a user makes the effort to disable it, that should serve as a social signal to others as to the type of person they are interacting with.
